lib/ini.js

```diff
@@ -9,7 +9,7 @@ const encode = (obj, opt = {}) => {
   opt.sort = opt.sort === true
   opt.whitespace = opt.whitespace === true || opt.align === true
   /* istanbul ignore next */
-  opt.platform = opt.platform || process?.platform
+  opt.platform = opt.platform || (typeof process === 'undefined' ? null : process.platform)
```

Comment:
Ternaries are very hard to visually parse so truthiness checks are typically the style we use.

process && process.platform.

Reply:
I'll try an alternative approach